wg-releases/major-release-process.md

```diff
@@ -0,0 +1,65 @@
+
+1) Run `/check-unmerged <branch-name>` in Slack and see if any PRs are returned
+2) Check open PRs to `master` on `electron/electron` and ensure that none of them carry the label `target/<branch-name>`
+
+If both tasks are complete, proceed with the final beta release.
+
+## Stable Preparation
+
+We need to ensure the following have owners and have been completed:
+
+* Blog Post
+  * This post should contain the following:
+    * An update for which versions are now supported, and the new types of support for each of those (security-only, full support, etc.)
+    * Link to timeline for subsequent major release
+    * A list of breaking changes in the current stable line
+    * References to any upcoming notable changes or deprecations
+* Tweet - a tweet about the latest stable line & blog post from the official `@electronjs` twitter account
+* Updates to [Supported Versions](https://github.com/electron/electron/blob/master/docs/tutorial/support.md#currently-supported-versions)
+* Timeline for subsequent [major release version](https://electronjs.org/docs/tutorial/electron-timelines)
+* [`node-abi`](https://github.com/lgeiger/node-abi) stable update
+  * Ensure that ABI version information is added for the first new stable version (Ex. `v6.0.0`)
+* [Spectron](https://github.com/electron-userland/spectron) update - a version of Spectron should be published compatible with the new stable release
+* [Chromedriver](https://github.com/electron/chromedriver) update - a version of Chromedriver should be published compatible with the version of Chromium being shipped in the new stable version
+
+## Stable Release Promotion
+
+The day before the publicly communicated date, we should begin the stable promotion process. This is to account for errors with processes out of our control, like CI errors or errors on npm.
+
+At approximately noon on that preceding day, a member of the Releases WG should trigger a release with Sudowoodo with the following parameters:
+
+```sh
+Release Branch: Stabilization Line (ie `6-0-x`)
+Release Type: `Stable`
+```
+
+## Immediately Following Release
+
+Immediately following the release, we need to ensure the following:
+
+* The [Electron Website](https://electronjs.org) reflects the latest stable version
+* The `latest` tag on npm correctly reflects the new latest major release line
+* The blog post has been successfully published to the [Electron Website](https://electronjs.org)
+
+## Subsequent Release Line Preparations
+
+After a new major line has been officially stabilized, we should begin to prepare for the beta process of the next line.
+
+For this, we need to ensure the following:
+
+* We have a new Node Module Version (NMV) for the line that follows what is about to become the new `master` major
+  * If we've just released `v6.0.0`, this means that `7-0-x` is about to become the beta line, and so we would need to procure a NMV for `8-0-x`
+* Sudowoodo is capable of handling betas for the new major line
```

Comment:
Should we link to the bit that needs updating.  i.e. https://github.com/electron/sudowoodo/blob/master/src/slack-manager.ts#L240-L265